### PR TITLE
chore(system-e2e): fix dockerfile cypress dist folder

### PR DESCRIPTION
**Pull request by busla**
Fixes reverted commit: https://github.com/island-is/island.is/pull/8195

## What
`Dockerfile.cypress` was copying wrong dist dir to image in CI build pipe.

## Why

Specify why you need to achieve this

## Screenshots / Gifs

Attach Screenshots / Gifs to help reviewers understand the scope of the pull request

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Formatting passes locally with my changes
- [ ] I have rebased against main before asking for a review

### CODE REVIEW

**Review by pshomov — APPROVED** [review]
LGTM

File: apps/system-e2e/src/support/commands.ts [hunk 1]
```diff
@@ -71,20 +73,23 @@ Cypress.Commands.add(
...
-  }
-})
+      cy.session('idsLogin', () => {
+        cy.session('cognitoLogin', () => cognitoLogin({ username, password }))
+        idsLogin({ phoneNumber, authUrl, baseUrl, urlPath })
+      })
+    } else {
+      cy.session('idsLogin', () => {
+        idsLogin({ phoneNumber, authUrl, baseUrl, urlPath })
+      })
```
Comment (kristofer-lp8f) [inline apps/system-e2e/src/support/commands.ts:89]: The if-statement can be refactored into `cy.session`

File: apps/system-e2e/src/lib/types.ts [hunk 2]
```diff
@@ -1,7 +1,23 @@
-import type { AuthUser, Discount } from '@island.is/air-discount-scheme/types'
+import type {
+  AuthUser,
```
Comment (kristofer-lp8f) [inline apps/system-e2e/src/lib/types.ts:2]:
```suggestion
  AuthUser as ADSUser,
```

File: apps/system-e2e/src/lib/types.ts [hunk 3]
```diff
@@ -45,20 +61,22 @@ type FakeUser = {
 
 interface IDSLogin {
   phoneNumber: string
+  authUrl: AuthUrl
   baseUrl: BaseUrl
   urlPath: string
```
Comment (kristofer-lp8f) [inline apps/system-e2e/src/lib/types.ts:66]:
```suggestion
  authUrl?: AuthUrl
  baseUrl?: BaseUrl
  urlPath?: string
```

File: apps/system-e2e/src/support/commands.ts [hunk 4]
```diff
@@ -1,6 +1,7 @@
 import { IDSLogin, CognitoCreds } from '../lib/types'
+import '@testing-library/cypress/add-commands'
```
Comment (kristofer-lp8f) [inline apps/system-e2e/src/support/commands.ts:2]: Won't this cause failures again?

File: apps/system-e2e/cypress.config.ts [hunk 5]
```diff
@@ -12,15 +13,15 @@ const getEnvironmentUrls = (env: TestEnvironment) => {
     : { authUrl: AuthUrl.local, baseUrl: BaseUrl.local }
 }
 
+const globalTimeout = 10000
```
Comment (kristofer-lp8f) [inline apps/system-e2e/cypress.config.ts:16]: Some commands (e.g. un-cached `idsLogin`) take > 20 seconds

**Review by kristofer-lp8f — APPROVED** [review]
Looks good. Some comments.

File: apps/system-e2e/src/lib/types.ts (hunk 2 above)
Comment (busla) [reply apps/system-e2e/src/lib/types.ts]: updated

File: apps/system-e2e/src/lib/types.ts (hunk 3 above)
Comment (busla) [reply apps/system-e2e/src/lib/types.ts]: updated

File: apps/system-e2e/cypress.config.ts (hunk 5 above)
Comment (busla) [reply apps/system-e2e/cypress.config.ts]: updated

**Review by peturgq — APPROVED** [review]

**Pull request merged**